src/en/authors-charm-metadata.md

```diff
@@ -22,7 +22,7 @@ Here's a valid metadata file:
     description: |
       MongoDB is a high-performance, open source, schema-free document- oriented
       data store that's easy to deploy, manage and use. It's network accessible,
-      written in C++ and offers the following features:  
+      written in C++ and offers the following features:
```

Comment:
I think covered in a different review, we don't want to strip all the trailing spaces - some of them have meaning

Reply:
What are they used for? I have checked both versions and they look the same.

Reply:
two trailing spaces insert a `<br>`. It is often a neater way to do it that to include html in the source or to mess the format up with hard carraige returns (which can cause other problems in markdown). They are pretty much essential in lists, as its the only way to force a linebreak without ending the list. So, although many of them are just cruft, some are there on purpose. It's just easier to disable any trailing space deletion, or people get [upset about it](https://github.com/Microsoft/vscode/issues/1679)